src/components/widgets/StoicWidget/index.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from "react";
+import { useGetStoicQuote } from "./useGetStoicQuote";
+import { ReactComponent as GreekLogo } from "../../../assets/icons/greek.svg";
+import { ReactComponent as CloudLogo } from "../../../assets/icons/cloud.svg";
+import { ReactComponent as CloudAltLogo } from "../../../assets/icons/cloud-alt.svg";
+
+interface StoicData {
+  author: string;
+  quote: string;
+}
+
+export const StoicWidget = () => {
+  const [stoicData, setStoicData] = useState<StoicData[] | undefined>([]);
```

Comment:
I would choose more precise names to describe the data. Like "stoicQuotes" or just "quotes". Data is super generic.